phpunit.xml.dist

```diff
@@ -8,6 +8,9 @@
   <filter>
     <whitelist>
       <directory suffix=".php">src</directory>
+      <exclude>
+        <directory suffix=".php">src/Logging/V2</directory>
```

Comment:
Looks good. We can exclude V10+ and betas with a trailing star:
`<directory suffix=".php">src/*/V[0-9]*</directory>`